src/FunnelChart/FunnelChart.uni.driver.d.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseUniDriver } from 'wix-ui-test-utils/unidriver';
+
+export interface FunnelChartUniDriver extends BaseUniDriver {
+  getItemsCount(): Promise<number>;
+  getValueAt(index: number): Promise<string>;
+  getLabel(index: number): Promise<string>;
```

Comment:
```suggestion
  getLabelAt(index: number): Promise<string>;
```
and fix in the actual code as well